src/cloudant/errors.py

```diff
@@ -32,8 +35,17 @@ def __init__(self, msg, code=None):
 
 class CloudantArgumentError(CloudantException):
     """
-    Exception class used to indicate an invalid argument being passed
-    to a CouchDB API
+    Provides a way to issue Cloudant Python client library specific exceptions
+    that pertain to invalid argument errors.  A CloudantArgumentError object is
+    instantiated with a message and optional code where the code defaults to
+    400.
+
+    Note:  The intended use for this class is internal to  the Cloudant Python
```

Comment:
Not picky at all.  Exactly what I'm looking for.  Fixed in 143d35fb18656c7d87e81763769c86f5bcd2360b.